__tests__/index.js

```diff
@@ -177,6 +182,52 @@ describe('pass options', () => {
       )
     ).toBe(true);
   });
+
+  test('should not rewrite entry file when skipRewritingSource is true', async () => {
+    const entryFile = 'index.js';
+
+    const prettierOptions = { tabWidth: 8 };
+
+    const files = {
+      [entryFile]: `${'very().'.repeat(20)}long("chaining")`,
+    };
+
+    const mockCheckResult = jest.fn();
+
+    const webpackConfiguration = getWebpackConfigWithRules([
+      {
+        test: /\.js$/,
+        use: [
+          {
+            loader: checkOutputLoader,
+            options: {
+              checkResult: mockCheckResult,
+            },
+          },
+          {
+            loader,
+            options: Object.assign({}, prettierOptions, {
+              skipRewritingSource: true,
+            }),
+          },
+        ],
+      },
+    ]);
+
+    const testFiles = await prepare(webpackConfiguration, files, entryFile);
+    const entryPath = Object.keys(testFiles)[0];
+    const entryContent = getContent(entryPath);
+    // entry file is not processed
+    expect(prettier.check(entryContent)).toBe(false);
```

Comment:
https://prettier.io/docs/en/api.html#prettierchecksource-options
I think, you should pass `prettierOptions` here too